logger.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"golang.zx2c4.com/wireguard/device"
+)
+
+// Use device package logger as the global logger for the application.
+// device.Logger is effectively a collection of standard logging library loggers
+var logger *device.Logger
+
+// logLevel global var, defaults to info
+var logLevel = device.LogLevelInfo
+
+// Sets the logLevel global variable. Needs to be called before the
+// initialisation of loggers
+func setLogLevel(level string) {
+	switch level {
+	case "debug":
+		logLevel = device.LogLevelDebug
+	case "info":
+		logLevel = device.LogLevelInfo
+	case "error":
+		logLevel = device.LogLevelError
+	}
```

Comment:
Do we want to catch an invalid value here and notify the user?
Silently ignoring a wrong value and using the default level isn't too bad in my opinion, but I thought I'd mention this.

Reply:
ouch, darwin! that's why I didn't catch this on build/test, thanks!